common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -673,9 +673,13 @@ else if (isBspProject(project)) {
     }
   }
 
+  public static boolean isBspModule(Module module) {
+    return ExternalSystemApiUtil.isExternalSystemAwareModule(PantsConstants.BSP_SYSTEM_ID, module);
+  }
+
+
   public static boolean isBspProject(Project project) {
-    ProjectSystemId id = ProjectSystemId.findById("BSP");
-    return id != null && ExternalProjectUtil.isExternalProject(project, id);
```

Comment:
why was that wrong?

Reply:
afaik scala plugin just does not set this any more